src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/TakeCardStyles.kt

```diff
@@ -18,40 +19,78 @@ class TakeCardStyles : Stylesheet() {
         val content by cssclass()
         val takeNumberLabel by cssclass()
         val timestampLabel by cssclass()
+        val defaultButton by cssclass()
+        val progressBar by cssclass()
+        val defaultGreen : Color = c("#58BD2F")
 
-        fun playIcon() = MaterialIconView(MaterialIcon.PLAY_CIRCLE_OUTLINE, "30px")
-        fun pauseIcon() = MaterialIconView(MaterialIcon.PAUSE_CIRCLE_OUTLINE, "30px")
+        fun pauseIcon() = MaterialIconView(MaterialIcon.PAUSE, "30px")
         fun deleteIcon() = MaterialIconView(MaterialIcon.DELETE, "25px")
         fun badgeIcon() = MaterialDesignIconView(MaterialDesignIcon.CREATION, "18px")
+        fun playIcon() = MaterialIconView(MaterialIcon.PLAY_ARROW, "25px")
+        fun editIcon() = MaterialIconView(MaterialIcon.EDIT, "25px").apply { fill = defaultGreen }
+        fun draggingIcon() = ImageLoader.load(
+                ClassLoader.getSystemResourceAsStream("images/baseline-drag_indicator-24px.svg"),
+                ImageLoader.Format.SVG
+        )
     }
 
     init {
         defaultTakeCard {
-            minWidth = 250.px
+            minWidth = 348.px
             maxWidth = minWidth
-            minHeight = 100.px
+            minHeight = 200.px
             maxHeight = minHeight
-            backgroundRadius += box(10.px)
+            backgroundRadius += box(5.px)
             badge {
                 backgroundRadius += box(0.px, 10.px, 0.px, 10.px)
                 padding = box(8.px)
                 iconStyle {
                     fill = Color.WHITE
                 }
             }
-            padding = box(0.px)
+            padding = box(5.px)
             content {
                 padding = box(10.px)
             }
             takeNumberLabel {
-                fontSize = 20.px
+                fontSize = 16.px
+                graphicTextGap = 7.5.px
+                fontWeight = FontWeight.BOLD
             }
             timestampLabel {
                 fontSize = 12.px
+                fontWeight = FontWeight.LIGHT
+                fontStyle = FontPosture.ITALIC
+                textFill = Color.LIGHTGRAY
+                padding = box(2.5.px)
             }
             button {
                 backgroundColor += Color.TRANSPARENT
             }
+            defaultButton {
+                minHeight = 40.px
+                minWidth = 150.px
+                borderRadius += box(5.0.px)
+                backgroundRadius += box(5.0.px)
+                borderColor += box(Color.LIGHTGRAY)
+                borderWidth += box(0.5.px)
+                effect = DropShadow(1.0,2.0,2.0,Color.LIGHTGRAY)
+                backgroundColor += Color.WHITE
+            }
+        }
+
+        progressBar {
+            track {
+                backgroundColor += Color.LIGHTGRAY
+                minHeight = 40.px
+                backgroundRadius += box(5.0.px)
+            }
+            bar {
+                backgroundColor += c("#0094F0")
```

Comment:
can this color be stored somewhere?

Reply:
i could move it to the top of the stylesheet. main thing is i just wanted it to be contained in this stylesheet because it has default values for this specific control

Reply:
I more meant our app colors theme thing

Reply:
I think this color is already in our AppColors. I just didn't want to put it here per say because i thought we didn't want our controls to be dependent on anything from the app directory?